cvat/apps/lambda_manager/views.py

```diff
@@ -335,62 +415,50 @@ def mandatory_arg(name: str) -> Any:
         response = self.gateway.invoke(self, payload)
 
         response_filtered = []
-        def check_attr_value(value, func_attr, db_attr):
+
+        def check_attr_value(value, db_attr):
             if db_attr is None:
                 return False
-            func_attr_type = func_attr["input_type"]
+
             db_attr_type = db_attr["input_type"]
-            # Check if attribute types are equal for function configuration and db spec
-            if func_attr_type == db_attr_type:
-                if func_attr_type == "number":
-                    return value.isnumeric()
-                elif func_attr_type == "checkbox":
-                    return value in ["true", "false"]
-                elif func_attr_type in ["select", "radio", "text"]:
-                    return True
-                else:
-                    return False
+            if db_attr_type == "number":
+                return value.isnumeric()
+            elif db_attr_type == "checkbox":
+                return value in ["true", "false"]
+            elif db_attr_type == "text":
+                return True
+            elif db_attr_type in ["select", "radio"]:
+                return value in db_attr["values"]
             else:
-                if func_attr_type == "number":
-                    return db_attr_type in ["select", "radio", "text"] and value.isnumeric()
-                elif func_attr_type == "text":
-                    return db_attr_type == "text" or \
-                           (db_attr_type in ["select", "radio"] and len(value.split(" ")) == 1)
-                elif func_attr_type == "select":
-                    return db_attr_type in ["radio", "text"]
-                elif func_attr_type == "radio":
-                    return db_attr_type in ["select", "text"]
-                elif func_attr_type == "checkbox":
-                    return value in ["true", "false"]
-                else:
-                    return False
+                return False
+
+        def filter_attributes(input_attributes, target_mapping):
+            attributes = []
+            for attr in input_attributes:
+                db_attr = next(filter(
+                    lambda x: x['name'] == attr['name'], target_mapping[item_label]['db_label'].attributespec_set.values()
```

Comment:
Seems like the `target_mapping[item_label]['db_label'].attributespec_set.values()` value should be passed as a parameter to this function, rather than `target_mapping`. Currently, it's getting reevaluated on each iteration of two nested loops.